configurations/audit.yaml

```diff
@@ -0,0 +1,5 @@
+append_scylla_yaml: |
+  audit: "table"
+  audit_categories: "DCL,DDL,AUTH,ADMIN"
+  audit_tables: "keyspace1.standard1,scylla_bench.test"
+  audit_keyspaces: "keyspace1,scylla_bench"
```

Comment:
What will be the impact of auditing keyspace1 for regular tests?
Both disk usage amplification and performance wise, as it may turn things slower and disruptions maybe can timeout because of it.
was this tested?

Reply:
> What will be the impact of auditing keyspace1 for regular tests?
> Both disk usage amplification and performance wise, as it may turn things slower and disruptions maybe can timeout because of it.

I also have same concern. 

> was this tested?

I started some short longevity firstly, will run longer jobs when the short one passed.

Reply:
The longevity-10g-3h had passed
- https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/byo-longevity-test/126/
- Test-id: 88d7507a-1766-4403-9e84-ef24a0d365c8

![image](https://user-images.githubusercontent.com/309708/103602948-4c395f00-4f48-11eb-9486-b28d932a804e.png)


Submitted more jobs:
- longevity-200gb-48h: https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/byo-longevity-test/127/ (running ..)
- longevity-50gb-4days: https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/byo-longevity-test/128/ (pending..)

Reply:
Hi @roydahan 

The Longevity-50gb-3days job (audit enabled) had executed 2.5 days, the nemesis, workloads, disk space all looks good.
I think it's safe to merge this PR (https://github.com/scylladb/scylla-cluster-tests/pull/3090) and https://github.com/scylladb/scylla-cluster-tests/pull/3091

Test job: https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/byo-longevity-test/130/
Test id: a4fec110-559e-4799-8b08-8d2534d8c67b

![image](https://user-images.githubusercontent.com/309708/104146408-9073a600-5405-11eb-819b-8859600593fa.png)
![image](https://user-images.githubusercontent.com/309708/104146415-949fc380-5405-11eb-9346-1779528dfc3f.png)